schemas/JSON/manifests/v1.2.0/manifest.singleton.1.2.0.json

```
@@ -54,6 +54,21 @@
         }
       }
     },
+    "Documentation": {
+      "type": "object",
+      "properties": {
+        "DocumentLabel": {
+          "type": [ "string", "null" ],

```

Documents should always be labeled
```suggestion
          "type": "string",
```